### PR TITLE
add: Login & Signup Pages styles

### PR DESCRIPTION
**Pull request by shilpe26**
Added Login and Signup Pages styles with features like:-
▪️On clicking the login button a login page will open and have the option of Signup from there also
login signup

### CODE REVIEW

File: src/pages/login/login.css [hunk 1]
```diff
@@ -22,31 +22,29 @@
...
-    display: flex;
+     display: flex;
     align-items: center;
     justify-content: center;
     background: rgba(255, 255, 255, .9);
-    position: fixed;
-    top: 0;
-    right: -105%;
+     position: fixed;
     z-index: 10000;
```
Comment (lomesshh) [inline src/pages/login/login.css]: No need of this much z-index reduce it

**Review by lomesshh — APPROVED** [review]
Remove commented code 
UI looks clean and beautiful

Rest all looks good to merge

File: src/pages/signup/Signup.jsx [hunk 2]
```diff
@@ -1,9 +1,56 @@
...
+					aria-label="true"
+				/>
+				<Link to="/">
+					<input
+						type="submit"
+						value="sign up"
+						className="btn"
+						aria-label="true"
+					/>
+				</Link>
```
Comment (thegeorgenikhil) [inline src/pages/signup/Signup.jsx]: Any reasons to use an input as a button?
It should be a ` button` element with `type="submit"`

**Review by thegeorgenikhil — APPROVED** [review]
Apart from the input submit, rest of the code looks fine.

File: src/pages/signup/Signup.jsx (hunk 2 above)
Comment (shilpe26) [reply src/pages/signup/Signup.jsx]: yes thank you👍

**Pull request merged**